crowbar_framework/app/models/service_object.rb

```diff
@@ -1025,11 +997,9 @@ def apply_role(role, inst, in_queue, bootstrap = false)
 
       unless delay.empty?
         Rails.logger.progress("Queuing the application of role #{role.name}")
-        # force not processing the queue further
-        in_queue = true
```

Comment:
what was this doing here?

Reply:
Umm, this actually might be needed so the `ensure` part gets properly triggered. Good catch!

Reply:
I know too little about how this whole function works, but if it was not removed on purpose then probably it's safer to leave it in :)